### ISSUE TITLE
Guide - For Proof read and use if you wish

### ISSUE THREAD

**Issue by scrubbless** [NPBruce/valkyrie#614]
Hi Guys, 

I have written a guide to using the editor (MoM specific), it has some more up to date screenshots and goes into more detail. I hope it helps.

Please feel free to download it, link it on your Wiki or pinch bits you like for your guides.

Also if you have any comments or would like to correct anything, let me know via BGG IM or email (scrubbless@gmail.com)

https://docs.google.com/document/d/1Gi5gTwKkw84Px0Lvk-bJ3Aml_vF3ZoljjB6a8FjWGt8/

**Comment by NPBruce** [comment]
Are you interested in wiki permissions to add these to the main wiki?

Some quick comments:

You can also update scenarios from the download page.


Quest component (see https://github.com/NPBruce/valkyrie/wiki/QuestIniQuest for most of this)

Thumbnails can also be in jpg format, which is probably more appropriate if there is no text or transparency.

For required expansions should state that RN and SM should not be used.

Max player number is 10 if you want to clarify that.

Yes difficulty is 0-1, not a bug.

Editor:
Click the + icon for an event will give you the option to create a {new} one.This doesn’t work for Add Components sadly. - Added as #616

Tokens:
Tokens work even if there is no text, all the event stuff happens you just don't see a dialog (this is why the camera moves).

Events:
Many events could trigger from EventStart, but Valkyrie doesn't make any promises on the execution order.  This may be fine if you have several different set up events (like setting up mythos variables).

Puzzles:
jpg images might be more appropriate in many cases.

"I have found that sometimes the default Cancel button does not appear for Item inspects. If you find that your puzzle box has two cancel buttons, just remove this second button."
Do you have any more info on this?  Can you please report as an issue?

UI:

This should mention that the layout and scaling will be different at 16x9 and 16x10 aspect ratios, Valkyrie supports both.  More on this here:
https://github.com/NPBruce/valkyrie/wiki/QuestIniUI

Custom Images:
"jpg’s work for some parts of the editor, but PNG’s are universal, so use them"

Can you please report any known issues with jpg?  It should work everywhere.

"it will only need to be a maximum of 1920 x 1080px"

This is a bit arbitrary, no reason you couldn't run this at 8K.  A high res jpg is generally better for quality/size than png

Audio:
Yes only ogg is supported (mostly because mp3 licensing). See also:
https://github.com/NPBruce/valkyrie/wiki/QuestIniEvent

"Tip - You can hit the Log button during your Save & Test to see what variables are set to. This is extremely useful when testing if events are working. We explained this when Testing the Mythos Events."

FYI you can do this in a normal game with ALT+d

Custom Monters:
"If you want the creature to do a variety of attack activations (like a ranged activation, horror activation and a melee activation) make this first event very simple and descriptive. "

Better to have no text at all - just use this to chain a random next event and add as many activation events as you like.

"Create a trigger event - there is a little bug here when creating an event from the activation. So create it first, then add it"
What is the bug?

**Comment by scrubbless** [comment]
> Custom Images:
> "jpg’s work for some parts of the editor, but PNG’s are universal, so use them"
> 
> Can you please report any known issues with jpg? It should work everywhere.

Yeh I will do, I had a couple of instances where I couldn't select a JPEG, but could a PNG. This was before I knew about the GitHub issue tracking. If I can reproduce i'll log an issue.


> "it will only need to be a maximum of 1920 x 1080px"
> 
> This is a bit arbitrary, no reason you couldn't run this at 8K. A high res jpg is generally better for quality/size than png

It was completely arbitrary, I was mainly trying to advise people against putting in massive 150mb images. Generally speaking people are going to be playing on a laptop or windows tablet so bigger than 1080p. I'll make it a tip and less "you must"


> "Tip - You can hit the Log button during your Save & Test to see what variables are set to. This is extremely useful when testing if events are working. We explained this when Testing the Mythos Events."
> 
> FYI you can do this in a normal game with ALT+d

Ohh handy, thanks!

> Custom Monters:
> "If you want the creature to do a variety of attack activations (like a ranged activation, horror activation and a melee activation) make this first event very simple and descriptive. "
> 
> Better to have no text at all - just use this to chain a random next event and add as many activation events as you like.

Ahh yeh, that makes sense, thanks!

I'll reproduce and submit it. But if you click -> on a Custom Monster event and click {New event} and rename it, when you go back to the Custom Monster it still links to the Event0 that was automatically created.

**Comment by scrubbless** [comment]
Sure i am happy to work on the wiki. Better to have it all in one place.

**Comment by NPBruce** [comment]
I have added you.  LordPyrex said he would work on adding this in #287 please collaborate.

**Comment by LordPyrex** [comment]
I think that the best way to implement this in the Wiki section is going to be to put your information into GitHubs text formatting. From there we can create a separate table of contents. Just let me know your thoughts on how you'd like to handle the incorporation of the information.

**Comment by scrubbless** [comment]
I don't mind, whatever is easiest.

**Comment by LordPyrex** [comment]
@scrubbless after looking into the transfer into GitHub, I think that it would be better to retain the Table of Contents by keeping this in your original document. I'd like to link to your Doc file, but I noticed that it is in Suggestion mode. I figured I'd ask you if you would want to switch it to Viewing Mode only, or did you want to leave it in suggestion mode so that others could send thoughts your way.
